lib/services/cloud-project-service.ts

```diff
@@ -24,10 +24,10 @@ export class CloudProjectService extends CloudService implements ICloudProjectSe
 		return path.join(options.projectDir, "cleanup-results");
 	}
 
-	public async cleanupProject(appIdentifier: string, projectName: string): Promise<ICleanupProjectResult> {
+	public async cleanupProject(cleanupProjectData: ICleanupProjectDataBase): Promise<ICleanupProjectResult> {
 		const cleanupTaskId = uuid.v4();
 		try {
-			const result = await this.executeCleanupProject(cleanupTaskId, appIdentifier, projectName);
+			const result = await this.executeCleanupProject(cleanupTaskId, cleanupProjectData.appIdentifier, cleanupProjectData.projectName);
```

Comment:
you can use destructuring on line 40:
```TypeScript
private async executeCleanupProject(cleanupTaskId: string, { appIdentifier: string, projectName: string }): Promise<ICleanupProjectResult>
```
and here call the method directly with cleanupProjectData:
```TypeScript
const result = await this.executeCleanupProject(cleanupTaskId, cleanupProjectData);
```